pytorch_lightning/loops/__init__.py

```diff
@@ -13,8 +13,11 @@
 # limitations under the License.
 
 from pytorch_lightning.loops.base import Loop  # noqa: F401
+from pytorch_lightning.loops.batch.training_batch_loop import TrainingBatchLoop  # noqa: F401
 from pytorch_lightning.loops.dataloader.dataloader_loop import DataLoaderLoop  # noqa: F401
-from pytorch_lightning.loops.dataloader.evaluation_dataloader_loop import EvaluationDataLoaderLoop  # noqa: F401
+from pytorch_lightning.loops.dataloader.evaluation_loop import EvaluationLoop  # noqa: F401
+from pytorch_lightning.loops.dataloader.prediction_loop import PredictionLoop  # noqa: F401
```

Comment:
I prefer chaining `__init__` imports:

```python
from pytorch_lightning.loops.dataloader import DataLoaderLoop, EvaluationLoop, PredictionLoop # noqa: F401
```

Reply:
sure, I just relied on isort to do the right thing.

Reply:
I don't think isort is that smart. It just checks the order and grouping but doesn't modify the import path